cmd/restic/cmd_backup.go

```diff
@@ -98,6 +100,8 @@ func init() {
 	f.StringVar(&backupOptions.FilesFrom, "files-from", "", "read the files to backup from file (can be combined with file args)")
 	f.StringVar(&backupOptions.TimeStamp, "time", "", "time of the backup (ex. '2012-11-01 22:08:41') (default: now)")
 	f.BoolVar(&backupOptions.WithAtime, "with-atime", false, "store the atime for all files and directories")
+	f.StringVar(&backupOptions.RootPrefix, "prefix", "", "apply a prefix to target paths")
```

Comment:
This needs a hint what kind of path you can supply as the prefix. Does it always need to be slash-separated? Or slash/backslash depending on the OS?

Reply:
Prefix should follow the same syntax as a normal path. It can be relative or absolute and does not need a final path separator because it's automatically appended.